### PR TITLE
Delay gnocchi client creation

### PR DESCRIPTION
**Pull request by sileht**
During the config phase, the python threading system is not yet
initialized. Making some lib not behaving correctly.

This moves the gnocchiclient setup to the init phase.

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now e8713ba158b8b117d120e5ffc02bf30a05b3883e

**Comment by jd** [comment]
This monotonic error seems to be new, but I'd prefer we fix first before merging anything else. I don't know why it does not work here. I imagine the same is going to happen on gnocchiclient gate.

**Comment by sileht** [comment]
Pull-request updated, HEAD is now a077c15ffa57ca38bfcdd3903bbe9c4275262a49

**Comment by sileht** [comment]
Pull-request updated, HEAD is now a077c15ffa57ca38bfcdd3903bbe9c4275262a49

**Comment by sileht** [comment]
Pull-request updated, HEAD is now b7e81461c6467b150648b34ca14e1b8310d2055c

File: collectd_gnocchi/__init__.py [hunk 1]
```diff
@@ -101,12 +116,9 @@ def init(self):
...
                 },
             })
-
-        self.values = []
-        self.batch_size = self.conf.get("batchsize", 10)
-
-        collectd.register_write(self.write)
-        collectd.register_flush(self.flush)
+        # FIXME(sileht): should be except ResourceTypeAlreadyExists
+        except exceptions.Conflict:
```
Comment (jd) [inline collectd_gnocchi/__init__.py]: This is ok now so just update the dependency on gnocchiclient?

Comment (sileht) [inline collectd_gnocchi/__init__.py]: Done

**Comment by sileht** [comment]
Pull-request updated, HEAD is now c2c8d9837d2340dd7d70c12f1c22a4cfb524afad

**Pull request merged**